### PR TITLE
fix: email embed for org

### LINKED ISSUE #13891: [CAL-3226] [CAL-3224] Embed in email not pulling any availability
Right now if you try to embed any event type in an email, it will show zero availability.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-3224](https://linear.app/calcom/issue/CAL-3224/embed-in-email-not-pulling-any-availability)</sub>

<sub>[CAL-3226](https://linear.app/calcom/issue/CAL-3226/[cal-3224]-embed-in-email-not-pulling-any-availability)</sub>

### PR DESCRIPTION
**Pull request by Udit-takkar**
## What does this PR do?

Fixes https://github.com/calcom/cal.com/issues/13891

https://threads.com/thread/34565959358/34565959361?s=a3bgrkotw3Je8iCfs29bpG

## Type of change

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Review by keithwillcode — DISMISSED** [review]
Let’s add unit tests for this logic. We already have a suite for this function

File: apps/web/test/lib/getSchedule.test.ts [hunk 1]
```diff
@@ -1255,5 +1259,59 @@ describe("getSchedule", () => {
...
+          startTime: `${plus1DateString}T18:30:00.000Z`,
+          endTime: `${plus2DateString}T18:29:59.999Z`,
+          timeZone: "Europe/London",
+          isTeamEvent: false,
+          orgSlug: "acme",
+          usernameList: [organizer.username],
+        },
+      });
+
+      expect(getScheduleRes).toBeDefined();
```
Comment (keithwillcode) [inline apps/web/test/lib/getSchedule.test.ts:1314]: We should make real assertions about the data returned.

**Review by keithwillcode — DISMISSED** [review]

File: apps/web/test/lib/getSchedule.test.ts (hunk 1 above)
Comment (keithwillcode) [reply apps/web/test/lib/getSchedule.test.ts:1314]: Yeah but there’s no reason to not add at least a couple assertions about the schedule that is returned so we verify if completely works

Comment (Udit-takkar) [reply apps/web/test/lib/getSchedule.test.ts]: Got it

**Review by keithwillcode — APPROVED** [review]
Code looks good. Not yet tested

**Pull request merged**